vnext/Microsoft.ReactNative/ReactInstanceSettings.h

```diff
@@ -178,7 +178,14 @@ struct ReactInstanceSettings : ReactInstanceSettingsT<ReactInstanceSettings> {
   IRedBoxHandler m_redBoxHandler{nullptr};
   hstring m_sourceBundleHost{};
   uint16_t m_sourceBundlePort{0};
+
+#if USE_HERMES
+  JSIEngine m_jSIEngineOverride{JSIEngine::Hernes};
+#elseif USE_V8
```

Comment:
You can build with both (and we soon will in desktop). Probably not a big deal regardless, but makes default behavior less predictable.